packages/teuchos/CMakeLists.txt

```diff
@@ -30,8 +30,16 @@ TRIBITS_ADD_DEBUG_OPTION()
 
 TRIBITS_ADD_SHOW_DEPRECATED_WARNINGS_OPTION()
 
-INCLUDE(CheckForAbiDemangle)
-CHECK_FOR_ABI_DEMANGLE(HAVE_GCC_ABI_DEMANGLE)
+#
+# The check needs to run code, so give cross-compiling users a bypass.
+#
+IF (DEFINED HAVE_GCC_ABI_DEMANGLE)
```

Comment:
@mhoemmen Would this be enabled by a cmake option?

Reply:
@jhux2 Yes.  The idea is that if you want to bypass the check, you just set the corresponding CMake option (in this case, `HAVE_GCC_ABI_DEMANGLE`) to the value that you want.  I learned this approach from @etphipp , back when he had to cross-compile for what I think was KNC.